### PR TITLE
Capabilities in Enviorment File for Allure reporting

### PR DESCRIPTION
**Pull request by ghost**
Management of the capabilities so it shows the relevant ones on the Allure report

### CODE REVIEW

**Review by raulgs97ntt — CHANGES_REQUESTED** [review]
Please, add a screenshot of Allure report to see the final result. Thanks!

File: src/main/java/utilities/AllureReport.java [hunk 1]
```diff
@@ -1,114 +1,123 @@
...
-          .append(LocalEnviroment.getAppIdentifier())
-          .append("</p>");
-      String apk = LocalEnviroment.getApp();
+
+      String appIdentifier = getAppIdentifier();
+      if (!FrontEndOperation.isNullOrEmpty(appIdentifier)) {
+        description = join(description, p(b("App Identifier: "), text(appIdentifier)));
+      }
+
+      String apk = getApp();
```
Comment (raunelgarcia) [inline src/main/java/utilities/AllureReport.java]: use directly getApp()

**Comment by alvarola17ntt** [comment]
> Please, add a screenshot of Allure report to see the final result. Thanks!


Test for Android in SauceLabs

Allure_android

Test for Chrome in SauceLabs

Allure_chrome

**Comment by alvarola17ntt** [comment]
> > Please, add a screenshot of Allure report to see the final result. Thanks!
> 
> Test for Android in SauceLabs
> 
> Allure_android
> 
> Test for Chrome in SauceLabs
> 
> Allure_chrome


Updated example for Android

Allure_android_2

**Review by raunelgarcia — APPROVED** [review]

**Comment by alvarola17ntt** [comment]
> Please, add a screenshot of Allure report to see the final result. Thanks!

Ok

**Review by raulgs97ntt — APPROVED** [review]

**Pull request merged**